public/pages/DetectorConfig/containers/Features.tsx

```diff
@@ -227,18 +237,34 @@ export class Features extends Component<FeaturesProps, FeaturesState> {
                 href={`${PLUGIN_NAME}#/detectors/${this.props.detectorId}/features`}
                 fill
               >
-                Add feature
+                Configure model
               </EuiButton>,
             ]}
           />
         ) : (
-          <EuiBasicTable
-            items={sortedItems}
-            columns={columns}
-            cellProps={getCellProps}
-            sorting={sorting}
-            onChange={this.handleTableChange}
-          />
+          <div>
+            <EuiFlexItem>
+              {getTitleWithCount('Features', featureNum)}
+              <EuiHorizontalRule margin="xs"/>
+              <EuiBasicTable
+                items={sortedItems}
+                columns={columns}
+                cellProps={getCellProps}
+                sorting={sorting}
+                onChange={this.handleTableChange}
+              />
+            </EuiFlexItem>
+            <EuiSpacer size="m"/>
+            <EuiFlexItem>
+              <EuiTitle size="s"><h3>Advanced settings</h3></EuiTitle>
+              <EuiHorizontalRule margin="xs"/>
+              <EuiBasicTable
```

Comment:
We use `EuiBasicTable` as we have multiple features, so we can put features in separate row.  Should we use the same UX design for `Advanced setting` ? How about use the similar design of `Detector configuration` ? Can you confirm with UX designer?

Reply:
I'll reach out to the UX designer to clarify. In the mockups, the `Advanced setting` section of this panel is stylistically identical to the features table in this panel.

Reply:
I synced up with the UX designer, and we will change the Advanced settings to be a content panel nested inside the Model configuration panel rather than a table.

Reply:
Cool, content panel should be good. Be sure to put both features and advanced setting to nested content panels.